ethereum/core/src/test/java/org/hyperledger/besu/ethereum/core/encoding/TransactionDecoderTest.java

```diff
@@ -93,4 +94,23 @@ void shouldDecodeWithHighNonce() {
     assertThat(transaction).isNotNull();
     assertThat(transaction.getNonce()).isEqualTo(MAX_NONCE - 1);
   }
+
+  @Test
+  void shouldCalculateCorrectTransactionSize() {
+    for (final String rlp_tx :
+        List.of(
+            FRONTIER_TX_RLP,
+            EIP1559_TX_RLP,
+            GOQUORUM_PRIVATE_TX_RLP,
+            NONCE_64_BIT_MAX_MINUS_2_TX_RLP)) {
+      System.out.println(rlp_tx);
```

Comment:
use a logger or remove the System.out.println

Reply:
removed